### PR TITLE
Plural routes by default

### PR DESCRIPTION
**Pull request by bertheyman**
This uses the plural of a modal for resource controllers by default to match the examples in the Laravel docs as discussed in #662 . Singular still remains possible via a config setting.

I checked the docs and could not find any references to the singular ones, so no need for changes there as far as I could see.

### CODE REVIEW

**Comment by jasonmccreary** [comment]
This is great. Please fix the linting errors.

Also, what about the views path? Again, probably not something documented, but I would imagine they should be plural by default too?

**Comment by bertheyman** [comment]
Thanks for checking!
I've fixed the linting error.

For the views path, the most official I could find are some examples from the docs.

Most use stuff like 'welcome.blade.php', but anything model related does use singular.

Example from the [docs](https://laravel.com/docs/10.x/routing#implicit-binding):
```php
// Route definition...
Route::get('/users/{user}', [UserController::class, 'show']);
 
// Controller method definition...
public function show(User $user)
{
    return view('user.profile', ['user' => $user]);
}
```

The [mailing section](https://laravel.com/docs/10.x/mail#configuring-the-view) also uses singular.

**TL;DR: current approach of Blueprint matches view folder conventions from the docs**.

**Comment by jasonmccreary** [comment]
Fair enough. Thanks for this!
